### ISSUE TITLE
Customize default bookshelf

### ISSUE THREAD

**Issue by angelicachao** [koodo-reader/koodo-reader#1069]
### I've searched the issue list, there is no similar features.

- [X] Yes

### If you have more than one feature, please open multiple issues to report respectively.

- [X] Yes

### Brief Introduction

Allow more custom options for the bookshelf that is opened by default

### Details

Currently, the bookshelf that is opened by default shows all the books at once, I'm suggesting to have more options in the settings, ex: all unsorted books or an option in each specific bookshelf to set it as default. Thank you!

**Comment by troyeguo** [comment]
Thanks for the advice, we will look into it.

**Comment by ClaudeRobbinCR** [comment]
Have you found any alternatives? 
None of the book readers appear to have this feature!

**Comment by angelicachao** [comment]
> Have you found any alternatives? None of the book readers appear to have this feature!
Sorry, I haven't been looking. Koodo works like a charm for me <3

**Comment by ClaudeRobbinCR** [comment]
> > Thanks for the advice, we will look into it.
> 
> Thank you! <3
> 
> > Have you found any alternatives? None of the book readers appear to have this feature!
> 
> Sorry, I haven't been looking since Koodo works like a charm for me already.

Yes it does for me as well BUT  a Reading list feature would be VERY helpful
